Tests/Tests/CommonUtilitySpec.swift

```diff
@@ -104,6 +104,38 @@ class CommonUtilitySpec: QuickSpec {
                 }
             }
 
+            context("CommonUtility.convertTriggerObjectToCustomEvent") {
+
+                it("will return Custom Event") {
+                    let attribute = TriggerAttribute(name: "att", value: "val",
+                                                     type: .string, operatorType: .isNotBlank)
+                    let trigger = Trigger(type: .event, eventType: .appStart, eventName: "app Start", attributes: [attribute])
+                    let result = CommonUtility.convertTriggerObjectToCustomEvent(trigger)
+                    expect(result).toNot(beNil())
+                }
+            }
+
+            context("CommonUtility.unlockResource") {
```

Comment:
Please rename the `CommonUtility.lock` context at the beginning so it's clear which test is testing what method. It might be `CommonUtility.lock(resourcesIn:for:)` for example